firefox-ios/firefox-ios-tests/Tests/XCUITests/ToolbarTest.swift

```diff
@@ -227,6 +227,10 @@ class ToolbarTests: FeatureFlaggedTestBase {
         // Swipe up to close the app does not work on iOS 15.
         if #available(iOS 16, *) {
             closeFromAppSwitcherAndRelaunch()
+            mozWaitForElementToExist(app.textFields[AccessibilityIdentifiers.Browser.AddressToolbar.searchTextField])
+            mozWaitForElementToExist(app.buttons[AccessibilityIdentifiers.Toolbar.settingsMenuButton])
+            mozWaitForElementToExist(app.staticTexts[AccessibilityIdentifiers.FirefoxHomepage.SectionTitles.topSites])
+            mozWaitForElementToExist(app.staticTexts[AccessibilityIdentifiers.FirefoxHomepage.SectionTitles.merino])
```

Comment:
If there's not enough time, the page may not open fully (but exists) and tabs button may not respond.